src/world.mojo

```diff
@@ -222,7 +223,7 @@ struct World:
     @always_inline
     fn _check_locked(self) raises:
         """
-        Checks if the world is locked, and panics if so.
+        Checks if the world is locked, and raises if so.
```

Comment:
Mid-term, it would probably be good to create a specific error type here.

Reply:
Agreed. Short-term mojo does not have specific errors (because they don't have inheritance)